src/components/table/partial-styles/_row-selection.scss

```diff
@@ -0,0 +1,81 @@
+#tabulator-table {
+    &.has-rowselector {
+        .limel-table--row-selector {
+            border-width: 0rem;
+            justify-content: center;
+
+            .tabulator-col-content {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                height: 100%;
+                line-height: 0;
+            }
+
+            .tabulator-col-title {
+                padding-left: 0;
+            }
+
+            // make the row selector column non-resizeable by hiding
+            // the resize handles between the frozen column and the
+            // column next to it
+            .tabulator-col-resize-handle {
+                display: none;
+            }
+            & + .tabulator-col:not(.tabulator-frozen),
+            & + .tabulator-cell:not(.tabulator-frozen) {
+                .tabulator-col-resize-handle.prev {
+                    display: none;
+                }
+            }
+
+            limel-checkbox {
+                transition: opacity 0.2s ease;
+                align-self: center;
+            }
+        }
+
+        .tabulator-row {
+            limel-checkbox {
+                opacity: 0.3;
+            }
+
+            &:hover {
+                .limel-table--row-selector {
+                    limel-checkbox {
+                        opacity: 1;
+                    }
+                }
+            }
+        }
+
+        &.has-selection {
+            .tabulator-row {
+                .limel-table--row-selector {
+                    limel-checkbox {
+                        opacity: 1;
+                    }
+                }
+            }
+        }
+    }
+}
+
+#tabulator-table {
+    // Below code overrides stupid-smartness of Tabulator. /Kia
```

Comment:
We can now remove these comments as well 😄

Reply:
Please do! :)